components/services/bat_ledger/bat_ledger_impl.cc

```diff
@@ -594,6 +594,22 @@ void BatLedgerImpl::UpdateMediaDuration(
   ledger_->UpdateMediaDuration(window_id, publisher_key, duration, first_visit);
 }
 
+void BatLedgerImpl::IsPublisherRegistered(
+    const std::string& publisher_id,
+    IsPublisherRegisteredCallback callback) {
+  // Move the callback into a `shared_ptr` so that it can be captured in the
+  // copyable lambda below.
+  auto shared_callback =
```

Comment:
personally I think if you want to use this pattern you should do a separate PR to change them all so they're consistent, but I'll leave it up to you